cpp/src/community/leiden.cu

```diff
@@ -30,8 +31,24 @@ std::pair<size_t, weight_t> leiden(raft::handle_t const &handle,
   CUGRAPH_EXPECTS(clustering != nullptr, "Invalid input argument: clustering is null");
 
   Leiden<GraphCSRView<vertex_t, edge_t, weight_t>> runner(handle, graph);
+  weight_t wt = runner(max_level, resolution);
 
-  return runner(clustering, max_level, resolution);
+  thrust::device_vector<vertex_t> vertex_ids_v(graph.number_of_vertices);
```

Comment:
If you don't need to initialize vector elements, rmm::device_uvector will be more efficient.

Reply:
Changed in the next push.